lib/buildpack/compile/installers/installer.rb

```diff
@@ -56,6 +56,10 @@ def should_install(_app_dir)
       false
     end
 
+    def version_file
```

Comment:
We decided to make this public so the helper methods in the SdkInfo module use this to determine the installed version of the .NET SDK